python/guide/module/gluon/activation.md

```diff
@@ -0,0 +1,241 @@
+
```

Comment:
1. I would store the images in images/* 
2. Naming scheme for ping files. Perhaps prefix with activation-
3. Are SVGs possible? Theyd' scale better?

Reply:
RE #1, I spoke with Aaron and we're going for a folder for each tutorial, inside of which is an `images` folder, and an `md` file of the same name as the folder.

Reply:
So this avoids issue in #2.